### ISSUE TITLE
[Request] Per item Stack Size/Disable general stack size modification

### ISSUE THREAD

**Issue by JasdanVM** [AverageAnime/StackSizeEdit#8]
Please allow us to disable the Maximum Stack Size modification to only use the  Stack Size Override feature to set some item stacks to arbitrary amounts.
 Let's say I want to set Cake and Soup Bowls to stack up to 16, Snowballs to 64, and Golden Apples/Carrots to 32.
Right now I would have to set the Maximum Stack Size to 64 and make a LONG list of Stack Size overrides, specially for all the items that aren't in `#c:not_stackable` and still aren't supposed to stack.
 I don't want to modify my maximum stack size, just edit a couple items, please make this change.

**Comment by AverageAnime** [comment]
I don't know how to disable the global change. In testing, adding a disable toggle will return items that stack to 64 to their proper default, but items that stack to 1 or 16 get changed to the max (2 billion).
However, I updated the mod to make what you describe much more reasonable.  Set the stack size to 64 and include "#c:not_stackable:1" and  "#c:16_stackable:16" in your override list. This will put it all the stack sizes back to default.  From there, you can add specific items/tags as you wish. Place them above the default overrides if it is an item that normally does not stack to 64.

**Comment by JasdanVM** [comment]
> "#c:not_stackable:1" and "#c:16_stackable:16"

I'm pretty sure this still is going to cause me some issues, perhaps some I won't even be aware off...
Because Cake, all cakes, which aren't supposed to stack as far as I know are getting to 64.
And the items I installed this for in the first place, the "Pick Your Poison" mod Darts, don't stack originally, but also are at  64 stack size when I set the maximum at that.
That's what I meant with:

> items that aren't in #c:not_stackable and still aren't supposed to stack.

Would there be a way to exclude all items that don't usually stack to 64 from the global value without using tags?
I mean something like this:
A toggle, that when enabled makes it so only items that would normally stack to 64 get affected by the maximum dtack size value.
I also know some mods don't have the standard stacking sizes, like "Absolute ULTRACRAFT" that has some projectiles that stack to 25, very deliberately, and would get affected.
